examples/md-flexible/strongscaling.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-
+export LC_NUMERIC=en_US.UTF-8
```

Comment:
We will also need this in the `measurePerf.sh` script

Reply:
you will do this!